lib/node/rpc.js

```diff
@@ -625,6 +628,22 @@ class RPC extends RPCBase {
     return this.chain.tip.height;
   }
 
+  async getFilterCount(args, help) {
```

Comment:
add some tests for the rpc functions as well.

Reply:
Okay.